epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/AnnotationProcessor.java

```diff
@@ -143,12 +143,19 @@ private Set<TypeElement> getLoggedTypes(RoundEnvironment roundEnv) {
             annotatedElements.stream()
                 .filter(e -> e instanceof TypeElement)
                 .map(e -> (TypeElement) e),
-            // 2. All type elements containing a field or method with the @Logged annotation
+            // 2. All type elements containing a field or method with the @Logged annotation,
+            // or interfaces which have an implementing class with an @Logged annotation
```

Comment:
It'd be better to change `methodsToLog` in `LoggerGenerator` to look at superclass and interface methods. This has the side effect of making a plain interface loggable if it happens to have a loggable implementation, which is completely invisible to anyone looking at the interface declaration.

Reply:
That was the intention. Currently, the "logger file" generated by epilogue for said interface is blank(so there's no side effects). The idea is that if you put @Logged on a IO impl but not the IO, then fields will still be logged